sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -4,10 +4,15 @@
 - Renamed type `SentimentScorePerLabel` to `SentimentConfidenceScorePerLabel`.
 - In both `DocumentSentiment` and `SentenceSentiment` property `SentimentScores` has been renamed to `ConfidenceScores`.
 - In `LinkedEntity`, property `Id` has been renamed to `DataSourceEntityId`.
-- Added `DetectLanguageInput.None` for user convenience, instead of passing empty string to `CountryHint`.
 - Change wording in all documentation from `text inputs` to `documents`.
 - Properties `Length` and `Offset` have been renamed to `GraphemeLength` and `GraphemeOffset` for the `SentenceSentiment`,
 `CategorizedEntity`, `PiiEntity`, and `LinkedEntityMatch` objects, to make it clear that the offsets and lengths are in units of Unicode graphemes.
+- Unify `DocumentSentimentLabel` and `SentenceSentimentLabel` into the same type now called `TextSentiment`.
```

Comment:
Very tiny nitpick: "Unified `DocumentSentimentLabel` and `SentenceSentimentLabel` into `TextSentiment`"

Reply:
Thanks!